### PR TITLE
feat: wishlistscreen with functionality

### PR DESCRIPTION
**Pull request by ianuragsharma**

### CODE REVIEW

File: src/components/cartProduct/CartProduct.jsx [hunk 1]
```diff
@@ -1,84 +1,81 @@
...
-            Move to Wishlist
-          </button>
+import { useCart, useWishlist } from "../../contexts";
+import { Link } from "react-router-dom";
+const CartProduct = ({ cartItem }) => {
+  const { cartDispatch } = useCart();
+  const { wishlistState, wishlistDispatch } = useWishlist();
+  const favouratedItem = () =>
+    wishlistState.some((item) => item._id === cartItem._id);
+  return (
```
Comment (rohanmathur91) [inline src/components/cartProduct/CartProduct.jsx:9]: have some spacing

**Review by rohanmathur91 — APPROVED** [review]
- Good folder structure
- Code is easy to understand
- Suggestion: rather than naming a folder helper you can name that folder as utils also

**Review by JaanviNL — APPROVED** [review]
Code is Clean and easy to understand

File: src/components/cartProduct/CartProduct.jsx (hunk 1 above)
Comment (ianuragsharma) [reply src/components/cartProduct/CartProduct.jsx:9]: ok will take care of that

**Pull request merged**